scripts/package

```diff
@@ -20,24 +20,17 @@ for i in "${IMAGES[@]}"; do
   IMAGE=${REPO}/"$i":${TAG}
   DOCKERFILE="package/$i/Dockerfile${TEMP_DOCKER_SUFFIX}"
 
-  FINAL_IMAGES=()
   if [ "$OS" = "windows" ]; then
       for v in "${NANOSERVER_VERSIONS[@]}"; do
         # append the nano-server version to the image tag
         IMAGE=${REPO}/"$i":${TAG}-${v}
+        (
+            set -x
+            docker build --build-arg "ARCH=${ARCH}" --build-arg "OS=${OS}" --build-arg="NANOSERVER_VERSION=${v}" -f ${DOCKERFILE} -t ${IMAGE} .
+        )
         docker build --build-arg "ARCH=${ARCH}" --build-arg "OS=${OS}" --build-arg="NANOSERVER_VERSION=${v}" -f ${DOCKERFILE} -t ${IMAGE}  .
```

Comment:
No need to build twice.

```suggestion
        (
            set -x
            docker build --build-arg "ARCH=${ARCH}" --build-arg "OS=${OS}" --build-arg="NANOSERVER_VERSION=${v}" -f ${DOCKERFILE} -t ${IMAGE} .
        )
```

Reply:
Ah right, one is supposed to be a log message